service/src/test/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatchTest.java

```diff
@@ -149,6 +150,39 @@ void createAzureLandingZone_LandingZoneDoesNotExistsExceptionIsHandledAndStartsJ
     assertEquals(result.getLandingZoneId(), LANDING_ZONE_ID);
   }
 
+  @Test
+  void listAzureLandingZoneResources_TagPropagation() {
+    setupLandingZoneResources();
+    ApiAzureLandingZoneResourcesList response =
+        landingZoneApiDispatch.listAzureLandingZoneResources(BEARER_TOKEN, LANDING_ZONE_ID);
+
+    verify(landingZoneService, times(1))
+        .listResourcesWithPurposes(eq(BEARER_TOKEN), eq(LANDING_ZONE_ID));
+    assertNotNull(response);
+    assertNotNull(response.getResources());
+    assertEquals(2, response.getResources().size());
+    response
+        .getResources()
+        .forEach(
+            resGroup -> {
+              assertEquals(
+                  3,
+                  resGroup.getDeployedResources().size(),
+                  "deployed resources size for type " + resGroup.getPurpose());
+              resGroup
+                  .getDeployedResources()
+                  .forEach(
+                      res -> {
+                        assertNotNull(
+                            res.getTags(), "tags null for resource id " + res.getResourceId());
+                        assertEquals(
+                            2,
+                            res.getTags().size(),
+                            "tags size for resource id " + res.getResourceId());
+                      });
+            });
+  }
```

Comment:
new unit test specifically to check that tags are returned in the api response. I could have added these asserts to an existing test, but that felt like it broke a bit of test encapsulation.